src/pages/VenuePrivateRoom/VenuePrivateRoom.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import { useParams } from "react-router";
+
+import Room from "components/organisms/Room";
+import WithNavigationBar from "components/organisms/WithNavigationBar";
+
+import "./VenuePrivateRoom.scss";
+
+export interface VenuePrivateRoomParams {
+  venueId: string;
+  roomId: string;
+}
+
+export const VenuePrivateRoom: React.FC = () => {
+  const { venueId, roomId } = useParams<VenuePrivateRoomParams>();
+
+  return (
+    <WithNavigationBar>
+      <Room roomName={roomId} venueName={venueId} setUserList={() => {}} />
+    </WithNavigationBar>
+  );
+};
```

Comment:
Some of our other routes are using this as well. I'm just following the pattern. It's a reusable component and I thought it's whole idea was to be reused because it accepts `children` as a prop

Reply:
A lot of those other routes are legacy and things we are removing.

It is a reusable component, but that doesn't mean we necessarily want to be using it all over the place if we can architect things better than that.

My comment in https://github.com/sparkletown/sparkle/pull/1385#discussion_r632325217 isn't inherently a blocker on this. The choice of whether this is a top level page/etc should be made based on different reasoning, which it seems you provided in another comment thread in this review (that it isn't part of/related to a venue at all), which negates my comment in https://github.com/sparkletown/sparkle/pull/1385#discussion_r632324295 about it being in `TemplateWrapper`.